tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": ["dom", "dom.iterable", "esnext", "ES2021.String"],
```

Comment:
I was getting a TS error because of the usage of `.replaceAll` string method, which is not available before `ES2021` and is related to the lib compilerOptions https://dirask.com/questions/TypeScript-property-replaceall-does-not-exist-on-type-string-DZEr21